dipy/workflows/mask.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+from __future__ import division
+
+import inspect
+import logging
+import numpy as np
+
+from dipy.io.image import load_nifti, save_nifti
+from dipy.workflows.workflow import Workflow
+
+
+class MaskFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'mask'
+
+    def run(self, input_files, greater_than=0.2, less_than=np.inf, out_dir='',
+            out_mask='mask.nii.gz'):
+
+        """ Workflow for creating a binary mask
+
+        Parameters
+        ----------
+        input_files : string
+           Path to image to be masked.
+        greater_than : float
+            Default is 0.2.
```

Comment:
I guess this should not be an optional param after all, no idea where 0.2 came from when I originally did this.